clients/cloud/c/README.md

```diff
@@ -2,17 +2,25 @@
 
 Produce messages to and consume messages from a Kafka cluster using the C client [librdkafka](https://github.com/edenhill/librdkafka).
 
-
 # Prerequisites
 
 * [librdkafka](https://github.com/edenhill/librdkafka) installed on your machine, see [installation instructions](https://github.com/edenhill/librdkafka/blob/master/README.md#instructions).
+* Create a local file (e.g. at `librdkafka.config`) with configuration parameters to connect to your Kafka cluster, which can be on your local host, [Confluent Cloud](https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud), or any other cluster.  Follow [these detailed instructions](https://github.com/confluentinc/configuration-templates/tree/master/README.md) to properly create this file. 
```

Comment:
@laura-brouckman , @rspurgeon 

did we want to go with `$HOME/.confluent/librdkafka.config` instead of `librdkafka.config`?

Reply:
good catch, the rest of the time I use `$HOME/.confluent/librdkafka.config`, so I'll update this line as well